cypress/e2e/tests/migration/migration-waves/create_tracker.test.ts

```diff
@@ -35,7 +35,7 @@ describe(["@tier1"], "Testing the creation of a tracker in migration waves", fun
         application.create();
     });
 
-    it("Verify create tracker button is visible", function () {
+    it("Bug MTA-1337| Verify create tracker button is visible", function () {
```

Comment:
Replace
Bug MTA-1337| Verify create tracker button is visible

with this:
Bug MTA-1337:  Verify create tracker button is visible

We agreed that this is the format we would use for marking blocker bugs .

Reply:
I think some people are following this format "Bug MTA 1234| description" and some are using colon 
"Bug MTA 1234: description"

Reply:
Done